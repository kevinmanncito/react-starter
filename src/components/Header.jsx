import React from 'react';


class Header extends React.Component {
  render () {
    return (
      <header className="header">
        <h1>React</h1>
        <img src={require('../images/me.png')} />
      </header>
    );
  }
}

export default Header;
