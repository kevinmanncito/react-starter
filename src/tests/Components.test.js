import React from 'react'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'

import Header from '../components/Header'

configure({ adapter: new Adapter() })

describe('Components render correctly', () => {
  it('Header renders correctly', () => {
    const wrapper = shallow(
      <Header />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
