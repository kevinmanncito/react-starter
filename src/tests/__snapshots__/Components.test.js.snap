// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components render correctly Header renders correctly 1`] = `
<header
  className="header"
>
  <h1>
    React
  </h1>
  <img
    src="me.png"
  />
</header>
`;
